spec/std/uri_spec.cr

```diff
@@ -54,6 +54,21 @@ describe "URI" do
     end
   end
 
+  describe "#absolute?" do
+    it { URI.parse("http://www.example.com/foo").absolute?.should eq(true) }
```

Comment:
`eq(true)` -> `be_true`
`eq(false)` -> `be_false`

Reply:
Nobody cares

Reply:
Well, it's not that nobody cares, it's just that the review adds very little value. In my mind I become the author and think "Ugh, do I really have to open the computer, sit down, change that, do a commit, push, etc., just to have some code that behaves exactly the same and is in now way superior to the old code?". That's what usually drives me from continuing working on a PR (as an author).

I know `be_true` is more idiomatic, but it's equivalent to `eq(true)` and it can be changed later, if someone really wants to do that.

Reply:
Exactly what I meant in longer form, thanks for clearing that up @asterite (and I apologise for my harsh tone)